examples/secure-env-vars/custom-entrypoint.sh.tftpl

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+set -e
+
+mkdir -p ${mount_folder}
+chown 100 ${mount_folder}
+cat <<'EOF' > "${mount_folder}/${entrypoint_filename}"
+#!/bin/bash
+
+set -e
+
+fetch_secret() {
```

Comment:
Do you think we could improve this? As now there's a func called `fetch_secret` but it's only used to write something to the filesystem. The other secrets aren't fetched this way. Should it be a func then? Can't we write a 1 liner like the other `gcloud secrets versions access` we do?

Reply:
updated as requested